lib/rbs/types.rb

```diff
@@ -107,10 +107,17 @@ def with_nonreturn_void?
       class Bool < Base; end
       class Void < Base; end
       class Any < Base
+        def initialize(location:, todo: false)
+          super(location: location)
+          todo! if todo
+        end
+
         def to_s(level=0)
           @string || "untyped"
         end
 
+        private
```

Comment:
The `private` here is not required for the change to work but `todo!` shouldn't be called from clients.

This will impact the public API, maybe we should make a deprecation first?

Reply:
Yeah, can you keep the method and print a deprecation message?